infrastructure/src/main/java/org/corfudb/infrastructure/ManagementAgent.java

```diff
@@ -324,7 +326,7 @@ private void bootstrapPrimarySequencerServer() {
             } else {
                 log.info("Bootstrapped sequencer server at epoch:{}", layout.getEpoch());
             }
-        } catch (InterruptedException | ExecutionException e) {
+        } catch (Exception e) {
```

Comment:
Can't we come up with specific messages when we knew it was InterruptedException or ExecutionException? In other words, although, we are including the exception description with the message, is is possible that with this granularity we are losing some info about the problem?

Reply:
Should not swallow the interrupt.